.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
add the .vscode/ folder to gitignore and delete it from the repo please

Reply:
All set.